express-api/.dockerignore

```diff
@@ -0,0 +1,9 @@
+coverage/
+./dist
```

Comment:
Just to make it look a little more like lines 1 and 3
```suggestion
dist/
```

Reply:
Included suggested change in the code.